src/tree/SlotsTreeItem.ts

```diff
@@ -45,27 +46,23 @@ export class SlotsTreeItem extends AzExtParentTreeItem {
         }
 
         const client: WebSiteManagementClient = await createWebSiteClient([context, this]);
-        const webAppCollection: WebSiteManagementModels.WebAppCollection = this._nextLink ?
-            await client.webApps.listSlotsNext(this._nextLink) :
-            await client.webApps.listSlots(this.parent.site.resourceGroup, this.parent.site.siteName);
-
-        this._nextLink = webAppCollection.nextLink;
+        const webAppCollection: Site[] = await uiUtils.listAllIterator(client.webApps.listSlots(this.parent.site.resourceGroup, this.parent.site.siteName));
```

Comment:
As Brandon mentioned, the paging doesn't actually work so we can't implement loadMore as of now.  I guess we should track this, though I'm not entirely sure where

Reply:
We should create an issue on the SDK repository for this. Brandon mentioned creating an issue, not sure if he got to it.

Reply:
I'll include the issue anywhere in code where we need to implement load more

Reply:
Yeah, I anticipate the following tree items could be potential issues:

- Resource groups
- Storage blobs/file shares
- Database documents

At least in the context of extensions that we already have